### PR TITLE
Propostal to use relay experimental

### PR DESCRIPTION
**Pull request by thiagoleitedev**
About UserList, will we add routes+preloadQuery in preparation or preparedloadQuery within itself?

### CODE REVIEW

**Comment by renanmav** [comment]
How would you preload the query?

File: packages/web/src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,23 @@
 import React from 'react';
```
Comment (sibelius) [inline packages/web/src/App.tsx]: why not { Suspense }

File: packages/web/src/ErrorBoundary.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+type Props = {
+  children: React.ReactNode;
+};
+type State = any;
+
+export default class ErrorBoundary extends React.Component<Props, State> {
+  constructor(props: Props) {
```
Comment (sibelius) [inline packages/web/src/ErrorBoundary.tsx]: we can remove constructor

use

state = { error: null } 

instead

type State properly

File: packages/web/src/index.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import "core-js/stable";
-import "regenerator-runtime/runtime";
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```
Comment (sibelius) [inline packages/web/src/index.tsx]: nice

File: packages/web/src/index.tsx [hunk 4]
```diff
@@ -8,4 +8,4 @@ import App from './App';
 const root = document.createElement('div');
 document.body.appendChild(root);
 
-ReactDOM.render(<App />, root);
+ReactDOM.createRoot(root).render(<App />);
```
Comment (sibelius) [inline packages/web/src/index.tsx]: nice

File: packages/web/src/App.tsx (hunk 1 above)
Comment (thiagoleitedev) [reply packages/web/src/App.tsx]: Sorry. Done!

**Comment by thiagoleitedev** [comment]
@renanmav 
So, I thought of these 2 solutions:
- Do inside the UserList. It is simpler, but it would be inappropriate.
- Use routes - It would be closer to actual use as the issue-tracker.

What do you think?

**Comment by renanmav** [comment]
Both solutions are valid. I prefer to preload on mouse hover.

**Comment by thiagoleitedev** [comment]
@renanmav 
Great! Im going to create a package with an issue-tracker router abstraction and bring the routes. Thanks!

**Comment by renanmav** [comment]
Actually I'm already working on this [here](https://github.com/relayjs/relay-examples/pull/122) with Typescript ~flow fede~ \o/

**Comment by thiagoleitedev** [comment]
Wow. Very good!
Thank you very much. I will need a lot @renanmav

**Pull request merged**